src/Api/Tracking.php

```diff
@@ -121,4 +122,39 @@ private static function get_logins(array $params): array
 
         return $response;
     }
+
+    /**
+     * Get last logins filtered by days.
+     *
+     * @param array $params refers to request params
+     * @return array Get logins.
+    */
+    private static function get_content_created(array $params): array
+    {
+        global $wpdb;
+
+        $response = [];
+
+        $query = $wpdb->get_results(
+            $wpdb->prepare(
+                "SELECT count(ID) as total, post_type
+                FROM wp_posts
```

Comment:
To maintain compatibility with dynamic database prefixes, please use `$wpdb->posts` instead of hardcoding wp_posts. This ensures that the correct table name is used, even if the WordPress database prefix changes.